test/widgets/message_list_test.dart

```diff
@@ -123,31 +123,25 @@ void main() {
 
   group('presents message content appropriately', () {
     // regression test for https://github.com/zulip/zulip-flutter/issues/736
-    testWidgets('content in "Combined feed" not asked to consume insets (including bottom)', (tester) async {
-      const fakePadding = FakeViewPadding(left: 10, top: 10, right: 10, bottom: 10);
-      tester.view.viewInsets = fakePadding;
-      tester.view.padding = fakePadding;
-
-      await setupMessageListPage(tester, narrow: const CombinedFeedNarrow(),
-        messages: [eg.streamMessage(content: ContentExample.codeBlockPlain.html)]);
-
-      final element = tester.element(find.byType(CodeBlock));
-      final padding = MediaQuery.of(element).padding;
-      check(padding).equals(EdgeInsets.zero);
```

Comment:
Yeah, this interpretation makes sense me. I decided to remove the `MentionsNarrow` test, because I think they will be largely verifying the same code path. It is reasonable to check for the absence of compose box in different narrows, but we are more interested in testing the inset fix here.